docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -1,39 +1,23 @@
 [id="migrating-ceph-mds_{context}"]
 
-//:context: migrating-ceph-mds
-//kgilliga: This module might be converted to an assembly.
-
-= Migrating Ceph MDS
-
-In the context of data plane adoption, where the OpenStack services are
-redeployed in OpenShift, a TripleO-deployed Ceph cluster will undergo a
-migration in a process we are calling “externalizing” the Ceph cluster.
-There are two deployment topologies, broadly, that include an “internal” Ceph
-cluster today: one is where OpenStack includes dedicated Storage nodes to host
-OSDs, and the other is Hyperconverged Infrastructure (HCI) where Compute nodes
-double up as Storage nodes. In either scenario, there are some Ceph processes
-that are deployed on OpenStack Controller nodes: Ceph monitors, rgw, rdb, mds,
-ceph dashboard and nfs-ganesha.
-This document describes how to migrate the MDS daemon in case Manila (deployed
-with either a cephfs-native or ceph-nfs backend) is part of the overcloud
-deployment.
-
-== Requirements
-
-For this procedure, we assume that we are beginning with a OpenStack based on
-Wallaby and a Ceph Reef deployment managed by TripleO.
+= Migrating {Ceph} MDS to an external cluster
+//kgilliga: Can you please verify the accuracy of the title?
```

Comment:
It's not accurate because we're not moving to "an external cluster": it implies the existence of a new external cluster, but in reality we're migration to a new set of target nodes within the same cluster, so I would change this title to something different that fits better the purpose of this section.

Reply:
OK, I suggest: "Migrating Ceph MDS to new nodes within the existing Red Hat Ceph Storage cluster"

or "new target nodes", but I was trying to shorten the title.

Reply:
ok, this is fine by me, I think we captured the intent of the section!